
led_state_list:
  
  - name: "EMERGENCY"
    led_name: "all_leds"
    mode: "blink"
    color: [20, 0, 0]
    ms_on: 100
    ms_off: 100


  - name: "TURN_LEFT"
    led_name: "front_left_leds"
    mode: "shift"
    color: [0, 20, 0]
    direction: "left"
    speed: 500
    sleep: 500


  - name: "TURN_RIGHT"
    led_name: "front_right_leds"
    mode: "shift"
    color: [0, 0, 20]
    direction: "right"
    speed: 500
    sleep: 500


  - name: "FORWARD"
    led_name: "rear_right_leds"
    mode: "blink"
    color: [0, 20, 20]
    ms_on: 100
    ms_off: 100


  - name: "STOP"
    led_name: "all_leds"
    mode: "paint"
    color: [0 , 0, 20]


  - name: "MOVING"
    led_name: "rear_left_leds"
    mode: "blink"
    color: [0,20,0]
    ms_on: 900
    ms_off: 900
